data/icons/meson.build

```diff
@@ -0,0 +1,33 @@
+actioniconsdir = pkgdatadir / 'icons' / 'hicolor'
+appiconsdir = datadir / 'icons' / 'hicolor'
+
+foreach size : ['16x16', '22x22', '24x24', '32x32']
+  install_data([
+    size / 'actions/eom-image-collection.png',
+    size / 'actions/slideshow-play.png',
+  ], install_dir: actioniconsdir / size / 'actions')
+endforeach
+
+foreach size : ['16x16', '22x22', '32x32']
+  install_data([
+    size / 'actions/eom-plugin.png',
+  ], install_dir: actioniconsdir / size / 'actions')
+endforeach
+
+install_data('48x48/actions/slideshow-play.png', install_dir: actioniconsdir / '48x48' / 'actions')
+
+install_data([
+  'scalable/actions/eom-image-collection.svg',
+  'scalable/actions/eom-plugin.svg',
+  'scalable/actions/slideshow-play.svg',
+], install_dir: actioniconsdir / 'scalable' / 'actions')
+
+foreach size : ['16x16', '22x22', '24x24', '32x32']
+  install_data([
+    size / 'apps/eom.png',
+  ], install_dir: appiconsdir / size / 'apps')
+endforeach
+
+install_data(['scalable/apps/eom.svg'], install_dir: appiconsdir / 'scalable' / 'apps')
+
+meson.add_install_script(find_program('gtk-update-icon-cache'), '-f', '-t', appiconsdir, skip_if_destdir: true)
```

Comment:
https://mesonbuild.com/Gnome-module.html#gnomepost_install

You can do `gnome.post_install(gtk_update_icon_cache: true)` and it handles both finding the program for you (under a couple different names) and invoking it depending on destdir.

Reply:
Also you're already using `gnome.post_install` elsewhere in this PR. You can call it multiple times and use additional features.

Reply:
Good catch. Initially I targeted 0.51 so I wrote the install script myself as this requires 0.57. Then the target version moved around during development so I probably miss some potential features I could've used.

Reply:
Ok, updated

Reply:
@eli-schwartz can we resolve the conversation?

Reply:
Indeed we can, my suggestion was implemented.